scripts/chocolatey/coder.nuspec

```
@@ -22,13 +22,14 @@
     <title>Coder (Install)</title>
```

```suggestion
    <title>Coder</title>
```
Why do we have (Install) as suffix here?

---

It's a pretty standard thing, but we don't need it, I did it on accident

---

removed in f7d2695